src/main/java/com/google/cloud/tools/maven/GenRepoInfoFileMojo.java

```diff
@@ -23,14 +23,12 @@
 import java.util.logging.Logger;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
-import org.apache.maven.plugins.annotations.Execute;
 import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
 
 /** Generates repository information files for the Stackdriver Debugger. */
-@Mojo(name = "genRepoInfoFile")
-@Execute(phase = LifecyclePhase.PREPARE_PACKAGE)
+@Mojo(name = "genRepoInfoFile", defaultPhase = LifecyclePhase.PREPARE_PACKAGE)
```

Comment:
this one's fine where it is.

Reply:
As in, it should fork?

Reply:
no, sorry, I just dind't want there to be any confusion about changing the default phase, and then I made it more confusing. Your changes here are fine.